doc/sphinx/dg.rst

```diff
@@ -1,3 +1,4 @@
+=================
```

Comment:
The current online documentation does not yet correspond to the current pyhton branch. I havent checked if the overline changes sth but if so the doc of sphinx is wrong.